client/src/Movies/SavedList.js

```diff
@@ -1,19 +1,25 @@
-import React, { Component } from 'react';
-
+import React, { Component } from 'react'
+import { NavLink, Link } from 'react-router-dom';
 export default class SavedList extends Component {
-  constructor(props) {
-    super(props);
+  constructor(props){
+    super(props)
   }
 
-  render() {
+  render(){
     return (
       <div className="saved-list">
-        <h3>Saved Movies:</h3>
-        {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
-        ))}
-        <div className="home-button">Home</div>
+        <h3 >Saved Movies:</h3>
+        {this.props.list.map( movie => {
+          return (
+            <NavLink to={`/movies/${movie.id}`} key={movie.id} activeClassName="saved-active">
+              <span className="saved-movie">{movie.title}</span>
+            </NavLink>
+          )
+        })}
```

Comment:
When an arrow function consists of several lines of body, you can utilize implicit return by wrapping the code inside parentheses. Like this:
```
{this.props.list.map( movie => (
  <NavLink to={`/movies/${movie.id}`} key={movie.id} activeClassName="saved-active">
    <span className="saved-movie">{movie.title}</span>
  </NavLink>
)}
```